BD-1876: Add tips for cloning Canvases (#4778)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/managing_canvases/cloning_canvases.md

As another example, if a Canvas has a full step that has exception events and this full step also uses the delay filter "in" or "on the next", then it can't be cloned to Canvas Flow. However, if a Canvas step uses exception events with any other delay type, then the Canvas can be cloned to Canvas Flow.

## Recommendations

> In order to allow existing users to continue their user journey after you've cloned your original Canvas to Canvas Flow, you can add filters to your existing Canvas to prevent new users from entering the new Canvas.

Reviewer: I mildly tripped up over the repetition of "to", maybe "which" here at the end? 

```suggestion
To allow existing users to continue their user journey after you've cloned your original Canvas to Canvas Flow, you can add filters to your existing Canvas which prevent new users from entering the new Canvas.
```

If re-eligibility is off, add the filter "Entered Canvas Variation". If re-eligibility is on, these are the possible methods to consider to ensure that users don't enter the same Canvas twice.
- Update the existing Canvas to include a unique tag. For the new Canvas, add a filter "Last Received Message from Campaign or Canvas with Tag". This will prevent users from entering the Canvas twice after a specific entry date (total number of days after the last message is sent from the original Canvas plus the conversion window). 
- **The following method will consume data points.** Update the original Canvas to include a Braze-to-Braze webhook that triggers a custom attribute date timestamp upon entry. This attribute can be used to prevent users from entering the new Canvas after the specified date (total number of days after the last message is sent from the original Canvas plus the conversion window).

> For API-triggered Canvases, coorindate with your engineering team to ensure that these Canvases are using the new Canvas ID once the new Canvases are ready to launch.

Reviewer: ```suggestion
For API-triggered Canvases, coordinate with your engineering team to ensure that these Canvases are using the new Canvas ID once the new Canvases are ready to launch.
```
